app/assets/javascripts/angular/controllers/mobile-dashboard.js

```diff
@@ -0,0 +1,200 @@
+          "sender": "Germany",
+          "recipient": "UK",
+          "content": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tincidunt nulla vel dui vulputate, dignissim porta nisl euismod.",
+          "created_at": '2016-03-20T12:29:00.000Z'
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "conversation_partner": "USA",
+      "updated_at": '2016-03-20T11:15:00.000Z',
+      "messages": [
+        {
+          "sender": "Germany",
+          "recipient": "USA",
+          "content": "Hi UK we miss you.",
+          "created_at": '2016-03-20T10:20:00.000Z'
+        },
+        {
+          "sender": "USA",
+          "recipient": "Germany",
+          "content": "A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. A boy and his dog. ",
+          "created_at": '2016-03-20T11:15:00.000Z'
+        }
+      ]
+    },
+  ];
+
+
+
+  // API Call and Status Check Intervals.
+  var apiCall = function() {
+    $http.get('/api/mobile_dashboard_data').then(
+      function successCallback(response) {
+        // Hide the connection error if it begins working again.
+        // See the Error callback for more information.
+        $('#connection-error').hide();
+
+        var result = response['data']['result'];
+        $scope.terror = result['global_terror'];
+        $scope.controlMessage = result['timer']['control_message'];
+        $scope.round = result['timer']['round'];
+
+        $scope.messages = result['messages'];
+
+
+        // Set the News
+        if (result['news'].length > 0){
+          var newDate = (new Date(result['news'][0]['created_at']));
+          if (($scope.news.length == 0) || newDate.getTime() > $scope.lastUpdatedNews.getTime()) {
+            $scope.lastUpdatedNews = newDate;
+            $scope.news = result['news'];
+          }
+        } else {
+          if ($scope.news.length == 0) {
+            $scope.news = result['news'];
+          }
+        }
+        var nextRound = moment(result['timer']['next_round']);
+        if ($scope.nextRound.valueOf() != nextRound.valueOf()) {
+          $scope.nextRound = moment(nextRound);
+          $scope.roundDuration = $scope.nextRound.diff(moment(), 'seconds');
+          $scope.$broadcast('timer-set-countdown',  $scope.roundDuration);
+          $scope.$broadcast('timer-start');
+          $scope.$broadcast('timer-set-end-time',  $scope.roundDuration);
+        }
+      },
+      function errorCallback(response) {
+        // If we are having trouble with the API call, this will display a
+        // small icon in the top right corner of the screen to signify that
+        // there is a connection issue.
+        $('#connection-error').show();
+      });
+  }
+
+  $scope.getStatus = function() {
+    apiCall();
+  };
+
+  $scope.getStatus();
+
+  $interval(function() {
+    $scope.getStatus();
+  }, 3000);
```

Comment:
We can handle it right now.  And if it's really becoming an issue, I'll move updating the game into a a chron job and then cache the api response on a 3-5 second basis.